test/ReCaptchaTest.php

```diff
@@ -28,14 +30,6 @@ public function setUp()
         if (! getenv('TESTS_ZEND_CAPTCHA_RECAPTCHA_SUPPORT')) {
             $this->markTestSkipped('Enable TESTS_ZEND_CAPTCHA_RECAPTCHA_SUPPORT to test PDF render');
```

Comment:
I think this message needs to be different, as recaptcha has nothing to do with PDF...